apps/src/polyfills.js

```diff
@@ -2,6 +2,7 @@
  * "Pointer Events Polyfill" to support pointer events on Safari
  */
 import 'pepjs';
+import wickedGoodXpath from 'wgxpath';
```

Comment:
Can we conditionally import this, so it doesn't increase bundle size for other browsers?

Reply:
I don't know of anywhere else that we lazy-load javascript, simply converting this to a `require` call and putting that into a conditional doesn't actually take it out of the bundle. Webpack supports dynamic imports/lazy loading, but that would require some additional config wrangling.